elfpy/markets/base.py

```diff
@@ -10,6 +10,7 @@
 
 import elfpy
 import elfpy.agents.wallet as wallet
+from elfpy.time.time import BlockTime
```

Comment:
can you change this to the `import _ as _` syntax? e.g.
```
import elfpy.time.time as time
time.BlockTime()
```

also, since it's only used for type checking can you put it under a TYPE_CHECKING flag?

Reply:
yup